prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -252,18 +253,20 @@ public IsPotentialPaperEnrolleeReturnee(IEnrolleePaperSubmissionService enrollee
         public override async Task<bool> ProcessRule(Enrollee enrollee)
         {
             var paperEnrollees = await _enrolleePaperSubmissionService.GetPotentialPaperEnrolleeReturneesAsync(enrollee.DateOfBirth);
-            var potentialPaperEnrolleeGpid = await _enrolleePaperSubmissionService.GetLinkedGpidAsync(enrollee.Id);
-            var paperEnrolleeMatchId = -1;
-
-            // Check if there's a match on a birthdate in paper enrollees, get all the ones that have a match
 
             // If there is no match then we don't need to worry about this rule
             if (!paperEnrollees.Any())
             {
                 return true;
             }
 
-            // if yes and GPID is provided
+            var potentialPaperEnrolleeGpid = await _enrolleePaperSubmissionService.GetLinkedGpidAsync(enrollee.Id);
+            var paperEnrolleeMatchId = -1;
+            var paperEnrolleeIdsAsString = CreateIdsString(paperEnrollees);
```

Comment:
You can easily simplify this helper to a one line.
``` var paperEnrolleeIdsAsString = string.Join(", ", paperEnrollees.Select(e => e.Id)); ```

Reply:
simplified